commit c86caaeaf063d7e33f180b1d18f1b149a79a9ee1
Author: John Levon <john.levon@joyent.com>
Date:   2019-08-27T21:56:04+00:00 (6 weeks ago)
    
    OS-7960 need fixes for NULL as a pointer
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/mdb_v8/Makefile b/mdb_v8/Makefile
index 1aeb337..a35b8cd 100644
--- a/mdb_v8/Makefile
+++ b/mdb_v8/Makefile
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -23,7 +23,7 @@
 # 3. From one directory up, run tar cvzf mdb_v8-<ver>.tar.gz mdb_v8-<ver>
 #
 
-VER =	mdb_v8-1.3.0
+VER =	mdb_v8-1.4.1
 
 include ../Makefile.defs
 include ../Makefile.targ
@@ -39,11 +39,6 @@ CTF_LIBS = $(VER)/build/ia32/mdb_v8.so \
 CFLAGS += -gdwarf-2
 CFLAGS_ARCH += -gdwarf-2
 
-#
-# To avoid "error: left shift of negative value"
-#
-CFLAGS += -Wno-shift-negative-value
-
 #
 # There are a couple of bugs in the currently released version of mdb_v8
 # that do not support parallelism, thus we explicitly do not include any
diff --git a/mdb_v8/mdb_v8-1.3.0.tar.gz b/mdb_v8/mdb_v8-1.3.0.tar.gz
deleted file mode 100644
index 184f612..0000000
Binary files a/mdb_v8/mdb_v8-1.3.0.tar.gz and /dev/null differ
diff --git a/mdb_v8/mdb_v8-1.4.1.tar.gz b/mdb_v8/mdb_v8-1.4.1.tar.gz
new file mode 100644
index 0000000..c6feac6
Binary files /dev/null and b/mdb_v8/mdb_v8-1.4.1.tar.gz differ
